lib/fakes3/file_store.rb

```diff
@@ -278,11 +279,24 @@ def create_metadata(content,request)
       end
       return metadata
     end
-     private
-     def get_objects bucket_name, path
-       Dir[File.join(path, '*')].map do |filepath|
-         get_object(bucket_name, File.basename(filepath), nil)
-       end
-     end 
+    private
+    def get_objects bucket_name, path, root_path=nil
```

Comment:
Use 2 spaces for indentation and () in the method declaration like the other methods in the class.